CONTRIBUTING.md

```diff
@@ -1,90 +1,37 @@
 # Contributing
 
+> [!NOTE]
+> This is a brief guide to contributing to yfinance.
+> For more information See the [Developer Guide](https://ranaroussi.github.io/yfinance/development) for more information.
+
 ## Changes
 
 The list of changes can be found in the [Changelog](https://github.com/ranaroussi/yfinance/blob/main/CHANGELOG.rst)
 
 ## Running a branch
 
-### With PIP
-
-```bash
-pip install git+https://github.com/ranaroussi/yfinance.git@BRANCH
-```
-
-E.g.:
-
-```bash
-pip install git+https://github.com/ranaroussi/yfinance.git@feature/something
-```
-
-### With Git
-
-1: Download from GitHub:
-
-```bash
-git clone https://github.com/ranaroussi/yfinance.git
-```
-
-Or if a specific branch:
-
 ```bash
-git clone -b <BRANCH NAME> https://github.com/ranaroussi/yfinance.git
-```
-
-> [!NOTE]
-> Only do the next part if you are installing globally
->
-> If you are installing for 1 specific project, then you can skip this step
-> and just `git clone` in the project directory
-
-2: Add download location to Python search path
-
-Two different ways, choose one:
-
-`. Add path to PYTHONPATH environment variable
-1. Add to top of Python file: 
-```python
-import sys
-sys.path.insert(0, "path/to/downloaded/yfinance")
+pip install git+https://github.com/{user}/{repo}.git@{branch}
```

Comment:
As this is brief guide, the pip command should just work, so change to
> pip install git+https://github.com/ranaroussi/yfinance.git@dev  # dev branch

Still obvious how to install other branches.